### ISSUE TITLE
The view.es_paginator could be thread unsafe

### ISSUE THREAD

**Issue by vdboor** [myarik/django-rest-elasticsearch#11]
Since the `request` is assigned the paginator, just providing a `es_paginator` attribute at class level could cause thread safety errors. When code is written like:

```
class View(ListElasticAPIView):
    es_paginator = ElasticLimitOffsetPagination()
```

This could cause thread safety errors.
Instead, I'd recommend using the paginator-property approach that the DRF uses (including `get_paginator_class()` and `get_paginator()`).

**Comment by myarik** [comment]
Thanks 
The problem was fixed.
Closed via e79b774a44770e9cf88a3540a41ed9b06b5ae37e

**Comment by vdboor** [comment]
Thanks a lot!
